src/modules/news/ui/views/NewsDetail.tsx

```
@@ -61,118 +59,254 @@ export default function NewsDetail() {
+                  </button>
+                </div>
+
+                {/* Content */}
+                <div
+                  className="prose prose-lg prose-slate max-w-none
+                    prose-headings:text-slate-900 prose-headings:font-bold
+                    prose-p:text-slate-700 prose-p:leading-relaxed prose-p:mb-6
+                    prose-strong:text-slate-900 prose-em:text-slate-700
+                    prose-a:text-blue-600 prose-a:no-underline hover:prose-a:underline"
+                  dangerouslySetInnerHTML={{
+                    __html: htmlContent || news.content || "Không có nội dung",
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)